tests/framework/common.go

```diff
@@ -225,6 +226,7 @@ func registerFlags(td *OsmTestData) {
 	flag.StringVar(&td.ClusterName, "kindClusterName", "osm-e2e", "Name of the Kind cluster to be created")
 	flag.BoolVar(&td.CleanupKindCluster, "cleanupKindCluster", true, "Cleanup kind cluster upon exit")
 	flag.BoolVar(&td.CleanupKindClusterBetweenTests, "cleanupKindClusterBetweenTests", false, "Cleanup kind cluster between tests")
+	flag.StringVar(&td.ClusterVersion, "kindClusterVersion", "", "Kind cluster version, ex. v.1.20.2")
```

Comment:
empty means whatever kind has as default?

Reply:
yes, it will by default pick the latest Kind node, currently `v1.20.2`